src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double length, double width) {
+        super(color);
+        this.length = length;
+        this.width = width;
+    }
+
+    public double getLength() {
+        return length;
+    }
+
+    public double getWidth() {
+        return width;
+    }
+
+    @Override
+    public double getArea() {
+        return length * width;
+    }
+
+    @Override
+    public String draw() {
+        return "Figure: rectangle, area: "
+                + String.format("%.2f",getArea()) + " sq. units, length: "
```

Comment:
Instead of using `String.format()` to format the area, length, and width, consider storing these formatted values in local variables before constructing the return string. This will improve readability.